valohai_yaml/objs/task.py

```diff
@@ -0,0 +1,109 @@
+from enum import Enum
```

Comment:
Adding the annotations future will let you use more modern type syntax in this brand new file. (At some point we'll probably add that to the rest of the codebase.)

You can run something like `ruff --select=UP006,UP007 --python-version=py311 valohai_yaml/objs/task.py` then, to have ruff autofix the old-style `Optional`s etc. for you.

```suggestion
from __future__import annotations

from enum import Enum
```